recipes/pystan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pystan" %}
+{% set version = "2.14.0.0" %}
+{% set sha256sum = "3bed255ca7d35bbd4ed0d5f30470744d9a71b06c1fa8b88281053dd430b0b1b2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+  patches:
+    - patches/0001-Disable-boost-quadmath.patch
+
+build:
+  number: 0
+  # For now, due to Travis time limit, we restrict the build matrix.
+  skip: True  # [osx and (py36 or np!=111)]
+  features:
+    - vc9   # [win and py27]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+  detect_binary_files_with_prefix: False
+
+requirements:
+  build:
+    - msinttypes  # [win and py27]
+    - python
+    - setuptools
+    - numpy >=1.7
+    - cython >=0.22,!=0.25.1
+
+  run:
+    - python
+    - numpy >=1.7
```

Comment:
Yes, as far I can tell. The source code does not `cimport numpy` anywhere except in `pystan/stanfit4model.pyx` which is a template to compile users' models and it doesn't get compiled during the `pystan` build.